scripts/util.py

```diff
@@ -141,32 +148,40 @@ def _download_dxfuse_to_resources(top_dir, dxfuse_version):
     if not os.path.exists(dxfuse_exe):
         os.makedirs(dxfuse_dir, exist_ok=True)
         info("downloading dxfuse {} to {}".format(dxfuse_version, dxfuse_exe))
-        subprocess.check_call([
-            "wget",
-            "https://github.com/dnanexus/dxfuse/releases/download/{}/dxfuse-linux".format(dxfuse_version),
-            "-O",
-            dxfuse_exe])
+        try:
+            subprocess.check_call([
+                "wget",
+                "https://github.com/dnanexus/dxfuse/releases/download/{}/dxfuse-linux".format(dxfuse_version),
+                "-O",
+                dxfuse_exe])
+        except subprocess.CalledProcessError as e:
+            print(e.stdout)
+            print(e.stderr)
+            raise e
 
         os.chmod(dxfuse_exe, 0o775)
 
     return dxfuse_exe
 
 
-def _create_asset_spec(version_id, top_dir, language):
+def _create_asset_spec(version_id, top_dir, language, dependencies=None):
     # TODO: update to 20.04 - just waiting for staging to catch up to prod
+    exec_depends = [
+        {"name": "openjdk-8-jre-headless"},
+        {"name": "bzip2"},
+        {"name": "jq"}
+    ] + (dependencies or [])
     asset_spec = {
         "version": version_id,
         "name": "dx{}rt".format(language.upper()),
-        "title": "dx {} asset".format(language.upper()),
-        "release": "16.04",
+        "title": "dx{} asset".format(language.upper()),
+        "release": "20.04",
         "distribution": "Ubuntu",
-        "execDepends": [
-            {"name": "openjdk-8-jre-headless"},
-            {"name": "bzip2"},
-            {"name": "jq"}
-        ],
+        "execDepends": exec_depends,
         "instanceType": "mem1_ssd1_v2_x4",
-        "description": "Prerequisites for running {} workflows compiled to the platform".format(language.upper())
+        "description": "Prerequisites for running {} workflows compiled to the platform".format(language.upper()),
+        # workaround for DEVEX-1827
+        "excludeResource": ["/snap*"]
```

Comment:
It should work without the `excludeResource` since the new asset builder that ignores snap was published.

Reply:
fixed